ios/Podfile.lock

```diff
@@ -2552,7 +2560,7 @@ SPEC CHECKSUMS:
   react-native-airship: 38e2596999242b68c933959d6145512e77937ac0
   react-native-blob-util: 1ddace5234c62e3e6e4e154d305ad07ef686599b
   react-native-cameraroll: f373bebbe9f6b7c3fd2a6f97c5171cda574cf957
-  react-native-config: 5330c8258265c1e5fdb8c009d2cabd6badd96727
+  react-native-config: 5ce986133b07fc258828b20b9506de0e683efc1c
```

Comment:
Also here, Do we need this change?

Reply:
This was just a result of running `bundle install && bundle exec pod install` after bumping react-native-screens. Do you see a different diff when you run that locally?

Reply:
Yes, it found on my podfile.lock locally.